dark/templates/dark/tournament/tround/info.html

```diff
@@ -15,52 +15,72 @@
 {% endblock %}
 
 {% block content %}
-    <div class="m-5">
-        <p>Name: {{ tround.name }}</p>
-        <p>Tournament: {{ tround.tournament }}</p>
-        <p>Start Date: {{ tround.start_date }}</p>
-        <p>End Date: {{ tround.end_date }}</p>
-    </div>
 
-    <h3>Requirements:</h3>
-    <table class="table">
-        <thead>
-        <tr>
-            <th scope="col">Library</th>
-            <th scope="col">Version</th>
-        </tr>
-        </thead>
-        <tbody>
-        {% for req in requirements %}
-            <tr>
-                <td align="left">{{ req.name }}</td>
-                <td align="left">{% for item in req.specs %} {% for i in item %} {{ i }} {% endfor %} {% endfor %}</td>
-            </tr>
-        {% endfor %}
-        </tbody>
-    </table>
+    <b>Name:</b> {{ tround.name }}<br>
```

Comment:
Wiem że to tutaj było, ale wydaje mi się że lepiej dać "Round", czy "Round Name" zamiast "Name", bo nie do końca wiadomo o co chodzi. Opcjonalnie